experiment_scheduler/task_manager/task_manager_server.py

```diff
@@ -307,20 +315,16 @@ def has_idle_resource(self, request, context):
         return IdleResources(exists=self._resource_manager.has_available_resource())
 
 
-def serve():
+def serve(address):
     """run task manager server"""
+    server = grpc.server(futures.ThreadPoolExecutor(max_workers=10))
+    task_manager_pb2_grpc.add_TaskManagerServicer_to_server(
+        TaskManagerServicer(), server
+    )
+    server.add_insecure_port(address)
+    server.start()
+    server.wait_for_termination()
 
-    with futures.ThreadPoolExecutor(max_workers=10) as pool:
-        server = grpc.server(pool)
-        task_manager_address = ast.literal_eval(USER_CONFIG.get("default","task_manager_address"))
-
-        task_manager_pb2_grpc.add_TaskManagerServicer_to_server(
-            TaskManagerServicer(), server
-        )
-        server.add_insecure_port(task_manager_address[0]) # [TODO] set multiple task manager
-        server.start()
-        server.wait_for_termination()
-        print("Interrupt Occurs. Now closing...")
 
 if __name__ == "__main__":
-    serve()
+    serve("[::]:50051")
```

Comment:
reverse serve() method as it was. Newer code is even older version than deleted one